spec/game_runner_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+require_relative '../game_runner.rb'
+
+RSpec.describe GameRunner do
+  def grid_template
+    <<~GRID
+   __ __ __
+  |  |  |  |
+  |__|__|__|
+  |  |  |  |
+  |__|__|__|
+  |  |  |  |
+  |__|__|__|
+    GRID
+  end
+
+  describe '#run' do
+    it 'on start up, an empty grid is rendered on the screen' do
+      stdout = StringIO.new(' ')
+      @game_start = GameRunner.new(stdout)
+
+      game = @game_start.run
+
+      expect(game).to eq(grid_template)
```

Comment:
This `game` will go away and then you need to figure out a way to get that information from the StringIO that you passed into the class on initialize.

Reply:
That makes sense!!